### ISSUE TITLE
Request to add one paper

### ISSUE THREAD

**Issue by SakurajimaMaiii** [YuejiangLIU/awesome-source-free-test-time-adaptation#16]
Hi Yuejiang,

Thanks for maintaining this awesome list!

I would like to request to add our paper: Feature Alignment and Uniformity for Test Time Adaptation (CVPR 2023). 
arxiv: https://arxiv.org/abs/2303.10902
code: https://github.com/SakurajimaMaiii/TSD
This belongs to online test time adaptation where the model can only access online unlabeled test samples and pre-trained models on the training domains. We propose two novle loss functions in a feature alignment & uniformity perspective.

Thanks for considering this.
Shuai Wang

**Comment by YuejiangLIU** [comment]
Hi Shuai, thanks for sharing that. I've just added your paper to the list.
